pvp-site.cabal

```diff
@@ -1,14 +1,16 @@
-cabal-version:      2.2
+cabal-version:      3.0
 name:               pvp-site
 version:            1.0
-tested-with:        GHC == 8.0.2 || == 8.2.2 || == 8.4.4 || == 8.6.3
+synopsis:           The Hakyll site for pvp.haskell.org
+tested-with:        GHC == 9.4.8 || == 9.6.6 || == 9.8.2
 
 executable pvp-site
   main-is:          site.hs
   hs-source-dirs:   src/
-  build-depends:    base          ^>= 4.9.0.0 || ^>= 4.10.0.0 || ^>= 4.11.0.0 || ^>= 4.12.0.0
-                  , hakyll        ^>= 4.12.5.0
-                  , pandoc        ^>= 2.6
-                  , pandoc-types  ^>= 1.17.5.4
+  build-depends:    base          >= 4.17 && < 5
```

Comment:
ironically not strictly following the pvp guidance....

Reply:
Thx, I am going to fix this. We also cannot make `cabal check` run through, since the cabal file is missing some core metadata like the `license` information. (And I don't want to come up with a license for a project which isn't my own). This is why I had to invoke `haskell-ci` with the `--no-cabal-check` option.

Reply:
> (base <5 bound is essentially the same as no upper bound, except it makes cabal check quiet)

Not really. 

This depends on the contracts between maintainers and library users.